homeassistant/components/zha/strings.json

```diff
@@ -27,6 +27,10 @@
           "flow_control": "data flow control"
         }
       },
+      "verify_radio": {
+        "title": "Radio is not recommended",
+        "description": "The radio you are using ({name}) is not recommended and support for it may be removed in the future. Please see the Zigbee Home Automation integration's documentation for [a list of recommended adapters](https://www.home-assistant.io/integrations/zha/#recommended-zigbee-radio-adapters-and-modules)."
```

Comment:
Don't put URLs in translations, instead, add a placeholder to the translation and inject the URL in the flow context.

This prevents the URL from translated, and prevents the need to invalidate all translations if the URL needs updating.